CRM/Logging/Schema.php

```diff
@@ -170,7 +171,7 @@ public function __construct() {
 FROM   INFORMATION_SCHEMA.TABLES
 WHERE  TABLE_SCHEMA = '{$this->db}'
 AND    TABLE_TYPE = 'BASE TABLE'
-AND    TABLE_NAME LIKE 'log_civicrm_%'
+AND    (TABLE_NAME LIKE 'log_civicrm_%' $nonStandardTableNameString )
```

Comment:
So on IRC Eileen mentioned use-case of applying logging to a custom table (e.g. created via Drupal Schema -- say `mytable`). The name would still be mapped as `mytable` <=> `log_mytable`, but the original query wouldn't hit it.

It seems like it would be simpler to say `TABLE_NAME LIKE 'log_%'`, but I guess you're more likely to hit accidental collisions that way (where some third party has defined a `log_foo` table totally unrelated to Civi). Using this helper `getNonStandardTableNameFilterString()` mitigates that risk.